### ISSUE TITLE
Feature Request: Fixed Assets in BOM - "Staging"

### ISSUE THREAD

**Issue by Tropicalrambler** [frappe/erpnext#6914]
I have an example, where, in order to manufacture a "Gizmo", the BOM needs:

1 Gadget (will remain with finished product)
1 Widget (will remain with finished product)
1 Bin (WILL NOT remain with finished product)

Thus, the process calls for the assembly of the **Gadget** and the **Widget** to produce a **Gizmo** while they remain on a "**Bin**", a fixed asset. The **Bin** serves as a logical unit containing those assembly parts. Once the production order is done, upon posting the stock entries the **Bin** should be returned to a specific warehouse as designated in the BOM by the user.

It would seem that there is a need for three sections in the BOM
1. Operations - already included.
2. Materials - already included
3. Fixed Assets -  to be added

In conclusion I would like to propose the following:
1. Addition of a fixed assets section for each BOM.
2. Destination warehouse can be selected for each fixed asset upon closing the Production Order.

For example, let's say that the **Bin**, once the **Widget** and **Gadget** have been assembled into the **Gizmo**, has to be cleaned. One possible warehouse must be "Post production Bin cleaning". There could be a BOM called _Bin cleaning_, which upon completion transfers the fixed asset back to the _Bin storage_ warehouse, so it is ready for the next assembly.

Ideas, thoughts, suggestions appreciated!

I also posted here:
[https://discuss.erpnext.com/t/bom-for-a-product-where-few-items-a-fixed/12229/2](url)
